week44/나머지가 1이 되는 수 찾기_hyeinlee725.py

```diff
@@ -0,0 +1,6 @@
+def solution(n):
+    for i in range(1, n):
+        if n % i == 1:
+            answer = i
+            break
+    return answer

```

Comment:
```suggestions
return answer
```

`break` 이후에 answer를 반환하기 때문에 `break` 없이 바로 `return` 해도 될 것 같아요!